src/emulate.c

```diff
@@ -593,18 +597,57 @@ static void block_translate(riscv_t *rv, block_t *block)
         ir->tailcall = next_ir->tailcall;                                  \
     }
 
+bool parse_memset(riscv_t *rv)
+{
+    uint32_t memset_insn[] = {
+        0x00f00313, 0x00050713, 0x02c37e63, 0x00f77793, 0x0a079063, 0x08059263,
+        0xff067693, 0x00f67613, 0x00e686b3, 0x00b72023, 0x00b72223, 0x00b72423,
+        0x00b72623, 0x01070713, 0xfed766e3, 0x00061463, 0x00008067, 0x40c306b3,
+        0x00269693, 0x00000297, 0x005686b3, 0x00c68067, 0x00b70723, 0x00b706a3,
+        0x00b70623, 0x00b705a3, 0x00b70523, 0x00b704a3, 0x00b70423, 0x00b703a3,
+        0x00b70323, 0x00b702a3, 0x00b70223, 0x00b701a3, 0x00b70123, 0x00b700a3,
+        0x00b70023, 0x00008067, 0x0ff5f593, 0x00859693, 0x00d5e5b3, 0x01059693,
+        0x00d5e5b3, 0xf6dff06f, 0x00279693, 0x00000297, 0x005686b3, 0x00008293,
+        0xfa0680e7, 0x00028093, 0xff078793, 0x40f70733, 0x00f60633, 0xf6c378e3,
+        0xf3dff06f};
+    uint32_t tmp_pc = rv->PC;
+    for (int i = 0; i < 55; i++) {
```

Comment:
It seems that both prologue and epilogue are included in the above sequence, right?